app/indexers/file_set_indexer.rb

```diff
@@ -0,0 +1,13 @@
+class FileSetIndexer < CurationConcerns::FileSetIndexer
+  def generate_solr_document
+    super.tap do |solr_doc|
+      solr_doc[Solrizer.solr_name('service_file_digest', :symbol)] = digest_from_service_file_content
```

Comment:
I see it won't index a nil value field but yes I can add condition check for it.